### DISCUSSION TITLE
CMI & DeluxeMenus. Placeholders

### DISCUSSION THREAD

**Discussion by Swersik** [Zrips/CMI#2458] [Q&A]
**Question: **
Hi, Zrips! I use the DeluxeMenus plugin and want to use the placeholder %cmi_user_rank% in it, but it does not work, instead of the rank of the user, I just write %cmi_user_rank%. There is no CMI in the /papi list. Perhaps the reason is this? How to add CMI library to /papi list? I ask for help. 
In the console, I observe the following message:
"CMI is currently using a deprecated method to hook into PlaceholderAPI. Placeholders for that plugin no longer work. Please consult [Zrips] and urge them to update it ASAP"

**Comment by CloudeLecaw** [comment]
you have to report this to the deuluxemenue dev. the problem is not with cmi
